lutron-aurora/src/init.lua

```diff
@@ -0,0 +1,102 @@
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+local Level = zcl_clusters.Level
+local ZigbeeDriver = require "st.zigbee"
+local capabilities = require "st.capabilities"
+local device_management = require "st.zigbee.device_management"
+local utils = require 'st.utils'
+
+local PowerConfiguration = zcl_clusters.PowerConfiguration
+local Groups = zcl_clusters.Groups
+
+local LAST_LEVEL_EVENT = "LAST_LEVEL_EVENT"
+local DEVICE_GROUP = "DEVICE_GROUP"
+--server: Basic, PowerConfiguration, Identify, TouchlinkCommissioning, 0xFC00 
+--client: Identify, Groups, OnOff, Level, OTAUpgrade, TouchlinkCommissioning
+
+local do_configure = function(self, device)
+  device:send(device_management.build_bind_request(device, PowerConfiguration.ID, self.environment_info.hub_zigbee_eui))
+  device:send(device_management.build_bind_request(device, Level.ID, self.environment_info.hub_zigbee_eui))
+  device:send(PowerConfiguration.attributes.BatteryPercentageRemaining:configure_reporting(device, 30, 21600, 1))
+end
+
+local function added_handler(self, device)
+  device:emit_component_event(device.profile.components[1], capabilities.button.numberOfButtons({ value = 1 }))
+  device:emit_event(capabilities.button.supportedButtonValues({"pushed"}, {visibility = { displayed = false }}))
+  device:send(PowerConfiguration.attributes.BatteryPercentageRemaining:read(device))
+  device:emit_event(capabilities.button.button.pushed({ state_change = false }))
+  device:emit_event(capabilities.switchLevel.level(0, { state_change = false }))
+end
+
+local battery_perc_attr_handler = function(_, device, value, _)
+  device:emit_event(capabilities.battery.battery(utils.clamp_value(value.value, 0, 100)))
```

Comment:
per the spec this should be reported as half percents (e.g. 200 = 100%), but we have seen this incorrect behavior before. Just wanted to confirm.

Reply:
Thank you, I am seeing the value of 200 currently so that makes more sense